fix path in curl GET example

lemonez opened the pull request:
Path to file in curl example was missing directory name so I made it consistent with the others. On a related note, getting a file seems fundamentally different from getting a directory (which just lists its contents). In my opinion these should be separated into "Get a file" and "List directory contents" sections. This would mirror the other "list" endpoints like "List algorithm builds", etc. Let me know if you agree and I can make a pull request for that piece.

bnlb commented:
> Path to file in curl example was missing directory name so I made it consistent with the others. On a related note, getting a file seems fundamentally different from getting a directory (which just lists its contents). In my opinion these should be separated into "Get a file" and "List directory contents" sections. This would mirror the other "list" endpoints like "List algorithm builds", etc. Let me know if you agree and I can make a pull request for that piece.

That's a fantastic suggestion. Make a PR for that at your leisure and I'll get it into the following marketplace deploy.

bnlb reviewed (approved):
👍

lemonez commented:
Lovely, will do 👍🏼

lemonez commented:
I added a commit for this piece to separate the "list directory contents" functionality. The Python methods for local files didn't translate to local directories when I tested them, so I deleted those from the Python code part.
